test/core/address.wast

```diff
@@ -101,3 +101,25 @@
   )
   "i32 constant"
 )
+
+;; Test that load/store addreses are full int32 (or int64), and not OCaml int
```

Comment:
Nit: can you remove the "not Ocaml int" part? I think it neither makes sense nor is it relevant.

Reply:
Sure :)